specification/configuration.md

```diff
@@ -110,17 +110,29 @@ beyond the OpenTelemetry specification exist.
 
 #### [OpenTelemetry Environment Variable](https://github.com/open-telemetry/opentelemetry-specification/blob/f228a83e652e5cd3ba96b9f780b704ee7a7daa4c/specification/sdk-environment-variables.md)
 
-- `OTEL_RESOURCE_ATTRIBUTES`
-  - User MUST define [`service.name`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#service)
-    resource attribute.
-  - Distribution MUST log a warning when the `service.name` resource attribute is not set. The
-    warning message MUST clearly describe how to set the attribute or link to relevant documentation.
-    E.g.
+- `OTEL_SERVICE_NAME`
+  - Users MUST define a name for the service they are instrumenting. The
+    service name can either be defined using the `OTEL_SERVICE_NAME` or
+    `OTEL_RESOURCE_ATTRIBUTES` environment variable, or directly in their code.
+    If the user fails to define a service name the distribution MUST log a
+    warning. The warning message MUST clearly describe how to set the service
+    name or link to relevant documentation. E.g.
     ```
     service.name attribute is not set, your service is unnamed and will be difficult to identify.
-    set your service name using the OTEL_RESOURCE_ATTRIBUTES environment variable.
-    E.g. `OTEL_RESOURCE_ATTRIBUTES="service.name=<YOUR_SERVICE_NAME_HERE>"`
+    set your service name using the OTEL_SERVICE_NAME environment variable.
+    E.g. `OTEL_SERVICE_NAME="<YOUR_SERVICE_NAME_HERE>"`
     ```
+  - The service name value MUST be used for the
+    [`service.name`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#service)
+    resource attribute.
+  - If a service name is defined with an environment variable and in the
```

Comment:
I think that at least in Otel Go and .NET SDK it the other way

Reply:
Right. I pulled out the overlap with the OTel spec and will rely on the specification for assignment and precedence.